src/PrestaShopBundle/Resources/views/Admin/Module/Includes/action_menu.html.twig

```diff
@@ -31,47 +31,41 @@
   module.attributes.name
 %}
 
-{% if level > constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_READ') %}
   {% if url_active == 'buy' %}
   <div class="form-action-button-container">
     <a class="btn btn-primary btn-primary-reverse btn-block btn-primary-outline light-button module_action_menu_go_to_addons" href="{{ url }}" target="_blank">
       {{ 'Discover'|trans({}, 'Admin.Modules.Feature') }}
     </a>
   </div>
-  {% else %}
+  {% elseif urls|length %}
   <div class="btn-group form-action-button-container">
     <form class="btn-group form-action-button" method="post" action="{{ urls[url_active] }}">
       <button type="submit" class="btn btn-primary-reverse btn-primary-outline light-button module_action_menu_{{ url_active }}"
-          data-confirm_modal="module-modal-confirm-{{ name }}-{{ url_active }}" {% if (level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_UPDATE')) or (level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE') and (url_active not in ['configure', 'install', 'upgrade'])) %} disabled="disabled" {% endif %}>
+          data-confirm_modal="module-modal-confirm-{{ name }}-{{ url_active }}">
```

Comment:
Why did you delete the level checks?

Reply:
Because we should avoid logical checks in the templates.

The checks are now done in a single place, and we now only have the available actions in the templates. Before, we had to check in the template if we could display them.

Reply:
Ok, I wanted to make sure of that, thanks 🙂